ee/modules/030-cloud-provider-openstack/openapi/doc-ru-config-values.yaml

```diff
@@ -0,0 +1,83 @@
+type: object
+properties:
+  connection:
+    description: Параметры подключения к api cloud provider'a.
+    properties:
+      authURL:
+        description: OpenStack Identity API URL.
+      caCert:
+        description: |
+          если OpenStack API имеет self-signed сертификат, можно указать CA x509 сертификат, использовавшийся для подписи.
+          * Сертификат в PEM формате.
+      domainName:
+        description: имя домена.
+      tenantName:
+        description: |
+          имя проекта.
```

Comment:
For deckhouse docs, we use capital letters because the description will be rendered as a new paragraph, e.g.
![image](https://user-images.githubusercontent.com/32434187/156640266-d1a4b679-3f12-4bff-a955-7d0e47b0ba03.png)

Reply:
The same goes for the English doc.

Reply:
It was like a line-by-line copy/patse from docs. Fixed